### ISSUE TITLE
Lighthouse Accessibility giving a question mark Malfunction

### ISSUE THREAD

**Issue by AmandaJaneDesign** [GoogleChrome/lighthouse#15002]
When I run lighthouse, it is flashing this Chrome services page for like .2 seconds and it is messing up my scores. The Accessibility is getting a "?" Question mark. I can't figure this out and I need to test sites for work. Please help 

![IMG_1082](https://user-images.githubusercontent.com/53097906/233471820-36b27346-3363-469a-b853-3d69557fec14.PNG)
![Screenshot 2023-04-20 at 3 44 01 PM](https://user-images.githubusercontent.com/53097906/233471978-db20c5ba-597a-4a17-8221-4cccad79e3b9.png)

**Comment by TripleEquals** [comment]
Hi! i'm one of the Google developers for cloud.google.com. We're unable to run our a11y reports ever since 10.0.0 became the standard version for lightrider. I'm also experiencing the same behavior as noted above for every url i've tested on cloud.google.com. Can we escalate this priority?

**Comment by jazyan** [comment]
Talked to @TripleEquals offline, URL to repro: https://cloud.google.com/gcp

It seems like the `color-contrast` audit is failing, with the following error:
```
    "color-contrast": {
      "id": "color-contrast",
      "title": "Background and foreground colors have a sufficient contrast ratio",
      "description": "Low-contrast text is difficult or impossible for many users to read. [Learn how to provide sufficient color contrast](https://dequeuniversity.com/rules/axe/4.6/color-contrast).",
      "score": null,
      "scoreDisplayMode": "error",
      "errorMessage": "axe-core Error: `TypeError: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': Value is not of type 'long'.` - feature unsupported in your environment. Skipping color-contrast rule."
    },
```

Does having a `null` `color-contrast` score block calculating the overall a11y score?

**Comment by adamraine** [comment]
Thanks @TripleEquals @jazyan, I can reproduce this!

> Does having a null color-contrast score block calculating the overall a11y score?

Yes, since we can't calculate a score for this specific audit we can't get the overall a11y score. That being said, it might be worth treating axe-core errors like these as `score: 0` rather than `score: null`.

Either way, we should investigate and fix this specific error.

**Comment by connorjclark** [comment]
https://github.com/dequelabs/axe-core/issues/3918

**Comment by mhaneef50673** [comment]
@adamraine axe-core has merged the following PR to address this issue,

https://github.com/dequelabs/axe-core/issues/3918
PR: https://github.com/dequelabs/axe-core/pull/4103

Could you please check and do the needful?

**Comment by Maverick737800** [comment]
Thanks, but not sure what any of that means? I get this error:

Background and foreground colors have a sufficient contrast ratio Error!
Low-contrast text is difficult or impossible for many users to read. [Learn how to provide sufficient color contrast]

But I have made sure everything on the site is within the ratio. 

Thanks for your help again.

**Comment by connorjclark** [comment]
@Maverick737800 The current state is that we are working on pulling in a new release of axe that will fix this issue. It's been mentioned right above your comment.

**Comment by Maverick737800** [comment]
Yeah ok cool. Thanks a lot. Had no idea what this really meant as I'm still new to the game. Thanks.

**Comment by moulinette** [comment]
Hi,
Not sure if this can help, but I have the same issue: 
```
_CONTRAST
Background and foreground colors have a sufficient contrast ratio **Error**!
axe-core Error: `TypeError: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': Value is not of type 'long'.` - feature unsupported in your environment. Skipping color-contrast rule_
```

- This happens with Lighthouse 10.1.0 to 10.4.0 and linux system
- It works correctly when using windows system (`_Background and foreground colors do not have a sufficient contrast ratio_`)
- It was working fine on both systems with Lighthouse 9.3.0

Any idea how to fix it for linux?

**Comment by adamraine** [comment]
> Forcing the usage of axe-core@4.8.0 fixes the error but the Contrast 'Failing Elements' reported list is smaller when using linux than when using windows. It seems that Unicode string are not well handled on axe-core when using linux platform.

We just updated axe-core to 4.8.0 in the latest Lighthouse version 11.1. I can't reproduce this issue.

Please verify that the issue still happens on your end and fill out a new bug report if it does.

**Comment by trieu-hphuc96** [comment]
hi, it has the same issue for this url: https://vnggames.com
![image](https://github.com/GoogleChrome/lighthouse/assets/22952175/cdef2cfe-3575-4b62-8e33-038d3cc00752)
